types.cc

```diff
@@ -23,6 +23,7 @@
 #include <cmath>
 #include <chrono>
 #include <sstream>
+#include <stdexcept>
```

Comment:
Why add this line in this patch?

Reply:
How was this resolved?

Reply:
I'm removing myself it in the new PR based on this one. So resolving this conversation.